src/css/ui/components/action-menu.css

```diff
@@ -26,7 +26,11 @@
   gap: 0;
   padding: 0.375rem;
   animation: slide-down 0.2s var(--anim-elastic);
-  z-index: var(--layer-popover);
+  /* 
+    It must be greater than `workspace-minimap` z-index value. 
+    Incrementing by 1 fixes issue #76.
+  */
+  z-index: calc(var(--layer-tooltip) + 1);
```

Comment:
Hmmm I think we might want to introduce a new layer 🤔

Reply:
`--layer-dropdown` or `--layer-action-menu` ?

Reply:
Yeah, I think we want distinct layers in the design system. Numbers on a case by case basis is a slippery slope. The layers are overdue for a review anyway.

Reply:
Oh there's apparently already one (I must have forgotten), called `--layer-floating-controls` for this exact thing.